File: src/pipCoding/code_java/Ch3_FibonacciNumbersInN.java
package pipCoding.code_java;

import java.util.Scanner;

public class Ch3_FibonacciNumbersInN {

    public static void main(String[] args) {
        Scanner scn = new Scanner(System.in);
        int n = scn.nextInt();

        int a = 0;
        int b = 1;
        int c;

        for (int i = 0; i <= n; i++) {
            System.out.println(a);
            c = a + b;
            a = b;
            b = c;
        }
    }
}

File: src/pipCoding/code_kotlin/ch3_FibonacciNumbersInN.kt
package pipCoding.code_kotlin

import java.util.*


fun main() {
    val scn = Scanner(System.`in`)
    val n = scn.nextInt()
    var a = 0
    var b = 1
    var c: Int
    for (i in 0 until n) {
        println(a)
        c = a + b
        a = b
        b = c
    }
}